lib/matplotlib/animation.py

```diff
@@ -611,6 +611,12 @@ class FFMpegFileWriter(FFMpegBase, FileMovieWriter):
     This effectively works as a slideshow input to ffmpeg with the fps passed as
     ``-framerate``, so see also `their notes on frame rates`_ for further details.
 
+    Parameters
+    ----------
+    *args, **kwargs
+        All arguments are forwarded to `FileMovieWriter`. See
+        `FileMovieWriter` for a list of all possible parameters.
+
     .. _their notes on frame rates: https://trac.ffmpeg.org/wiki/Slideshow#Framerates
```

Comment:
This does not render correctly. https://output.circle-artifacts.com/output/job/59253c59-88ba-4521-9a5e-ae6dfbad4c92/artifacts/0/doc/build/html/api/_as_gen/matplotlib.animation.FFMpegFileWriter.html#matplotlib.animation.FFMpegFileWriter

Try putting the parameters after the link target.

```suggestion
   .. _their notes on frame rates: https://trac.ffmpeg.org/wiki/Slideshow#Framerates

    Parameters
    ----------
    *args, **kwargs
        All arguments are forwarded to `FileMovieWriter`. See
        `FileMovieWriter` for a list of all possible parameters.
```